basic_samples/Authentication/HybridFlow/DotNet/HybridFlow/LoopbackHttpListener.cs

```diff
@@ -17,7 +17,7 @@ public class LoopbackHttpListener : IDisposable
         public LoopbackHttpListener(int port, string path = null)
         {
             path ??= string.Empty;
-            if (path.StartsWith("/", StringComparison.OrdinalIgnoreCase)) path = path.Substring(1);
+            if (path.StartsWith("/", StringComparison.OrdinalIgnoreCase)) path = path[1..];
```

Comment:
is there a reason we are switching to this?  Substring still seems to be in dotnet 5 and I haven't seen this format used often

Reply:
I haven't either, but Visual Studio recommended using this new format (popped up as an "Information" build message).